.github/workflows/ci.yml

```
@@ -1,42 +1,39 @@
-    runs-on: ubuntu-latest
+  ci:
+    name: CI Checks
+    uses: couimet/github-actions/.github/workflows/typescript-ci-checks.yml@main
     permissions:
       contents: read
       pull-requests: write
+    with:
+      build-command: pnpm compile
+      codecov-upload: true
+      codecov-files: packages/*/coverage/lcov.info
+      codecov-token: ${{ secrets.CODECOV_TOKEN }}
```

actionlint (1.7.12)

[error] context "secrets" is not allowed here. available contexts are "github", "inputs", "matrix", "needs", "strategy", "vars". see https://docs.github.com/en/actions/learn-github-actions/contexts#context-availability for more details

(expression)